packages/esm-appointments-app/translations/he.json

```diff
@@ -0,0 +1,144 @@
+{
+  "actions": "פעולות",
+  "addNewAppointment": "הוספת תור חדש",
+  "addNewClinicDay": "הוספת יום קליניקה חדש",
+  "addPatientToQueue": "הוספת מטופל לתור",
+  "AddPatientToQueue": "הוספת מטופל לתור",
+  "all": "הכל",
+  "anotherVisitType": "התחלת סוג ביקור נוסף",
+  "appointmentCancelError": "שגיאה בביטול התור",
+  "appointmentCancelled": "התור בוטל",
+  "appointmentColor": "צבע התור",
+  "appointmentDateAndTime": "תאריך ושעת התור",
+  "appointmentForm": "טופס התור",
+  "appointmentFormError": "שגיאה בתזמון התור",
+  "appointmentHistory": "היסטוריית התורים",
+  "appointmentList": "רשימת התורים",
+  "appointmentMarkedAsMissed": "סומן בהצלחה כנעדר",
+  "appointmentMetrics": "מדדי התורים",
+  "appointmentMissed": "התור נעדר",
+  "appointmentMissedError": "שגיאה בסימון התור כנעדר",
+  "appointmentNotes": "הערות לתור",
+  "appointmentNowVisible": "התור כעת נראה בעמוד התורים",
+  "appointments": "תורים",
+  "appointmentScheduled": "התור נקבע",
+  "appointmentService": "שירות התור",
+  "appointmentServiceCreate": "שירות התור נוצר בהצלחה",
+  "appointmentServiceName": "שם שירות התור",
+  "appointmentsList": "רשימת התורים ריקה",
+  "appointmentStatusChange": "סטטוס התור שונה בהצלחה ל-",
+  "appointmentStatusTitleMessage": "סטטוס התור",
+  "backToSearchResults": "חזור לתוצאות החיפוש",
+  "calendar": "לוח שנה",
+  "cancel": "ביטול",
+  "cancelAppointment": "ביטול תור",
+  "cancelled": "בוטל",
+  "changeAppointmentStatus": "שינוי סטטוס התור",
+  "checkedIn": "נרשם נכנס",
+  "checkFilters": "בדוק את המסננים למעלה",
+  "checkIn": "הרשמה",
+  "chooseProvider": "בחירת ספק",
+  "chooseQueue": "בחירת תור",
+  "chooseService": "בחירת שירות",
+  "clinicalAppointments": "תורים קליניים",
+  "comments": "הערות",
+  "community": "קהילה",
+  "complete": "השלמה",
+  "completed": "הושלם",
+  "createAppointmentService": "יצירת שירותי תורים",
+  "daily": "יומי",
+  "date": "תאריך",
+  "dateAndTimeOfVisit": "תאריך ושעת הביקור",
+  "dateTime": "תאריך ושעה",
+  "day": "יום",
+  "defaulters": "מנעדרים",
+  "discard": "סגירה מבלי לשמור",
+  "doubleBooking": "שיבוץ תור כפול",
+  "doublePatientBooking": "קיים תור קיים עבור השירות ותאריך התור המצוין",
+  "downloadAppointmentList": "הורדת רשימת התורים",
+  "durationMins": "משך בדקות",
+  "editAppointment": "עריכת תור",
+  "emergency": "חירום",
+  "emptyIllustration": "איור נתונים ריק",
+  "encounters": "פגישות",
+  "encounterType": "סוג הפגישה",
+  "endTime": "שעת סיום",
+  "errorAppoinmentMetric": "",
+  "errorMessage": "שגיאה בביטול התור",
+  "exitAndChangeStatus": "שינוי סטטוס",
+  "facility": "מוסד",
+  "filter": "סינון",
+  "filterByServiceType": "",
+  "frequency": "תדירות",
+  "futureScheduledVisits": " {count, plural, one {{count} ביקור} two {{count} ביקורים} many {{count} ביקורים} other {{count} ביקורים}} scheduled for dates in the future",
```

Comment:
Shouldn't the "scheduled for dates in the future" part also be in Hebrew?

Reply:
Fixed. Thank you! I also noticed that other languages have the same issue in these places.